unit_tests/src/test_rcConnect.cpp

```diff
@@ -41,4 +42,19 @@ TEST_CASE("set_session_signature_client_side")
         CHECK(set_session_signature_client_side(&comm, nullptr, buf.size()) == SYS_INVALID_INPUT_PARAM);
         CHECK(std::strncmp(comm.session_signature, "", sig_size) == 0);
     }
+
+}
+
+TEST_CASE("rcDisconnect segfault")
```

Comment:
Let's remove segfault.

This can cover all `rcDisconnect` test situations moving forward.